tests/e2e/test_rosacli_upgrade.go

```diff
@@ -449,6 +504,25 @@ var _ = Describe("Describe/List rosa upgrade",
 			})
 	})
 
+func FindUpgradeVersion(v rosacli.VersionService, channelGroup string, hostedCP bool, clusterVersion string) (string,
```

Comment:
IMO, we should do the Y-stream upgrading in the process y->y+1->y+2. So we should find the latest y+1 version.

Reply:
Update PR according to our discussion, sorted upgrade path versions.